torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5984,6 +5995,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
     OpInfo('vstack',
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            sample_inputs_func=sample_inputs_hstack_dstack_vstack,
+           supports_forward_ad=True,
```

Comment:
Nice! `hstack` and `vstack` are indirectly supported because of `cat`.  You able to find out because of https://github.com/pytorch/pytorch/pull/58230/files#diff-d183f2afc51d6a59bc70094e8f476d2468c45e415500f6eb60abad955e065156R236 this line you added earlier right?

Not sure I see cross` being tested. Maybe you could modify `test_cross` in test_linalg and also method tests for cross

Reply:
Yes exactly. It will actually fail if I add support for something without setting the flag to True.

Let me update the cross test!